bears/xml2/XMLBear.py

```diff
@@ -50,17 +50,21 @@ class XMLBear:
     @staticmethod
     def create_arguments(filename, file, config_file,
                          xml_schema: path='',
```

Comment:
I also wonder if this might support a URL also?
If so, it should use `path_or_url`